### PR TITLE
Kurdistan tweaks

### PR DESCRIPTION
**Pull request by ccfox**
-Added in two new tags for South and Eastern Kurdistan to make event lines run smoother, rather than being mutually exclusive and intersecting.
-Added in a South Azerbaijan tag for a similar purpose.
-Gave the USSR some benefit to intervening in Iran by allowing them to get both (technically they have soldiers stationed in both anyhow, and military operations to retake them necessitated a Soviet withdraw).
-Changed (all, I hope) mentions of the KUR and AZB tag to correspond to the appropriate tag if needed.
-Added in a decision for the Kurdistans to unite with the main Kurdistan tag, or to become it - as well as events for South Azerbaijan to do the same with Azerbaijan.

### CODE REVIEW

**Comment by Outta-my-way-burrocrat** [comment]
Would you like a Kurd oob file I've been using? Also will you be adding any Kurdish pops to west Iran?

**Comment by ccfox** [comment]
Sure, I haven't altered the OOBs because I'm not good at setting them up. Also, unless the populations there are incorrect - I believe there already are Kurdish pops in the areas that there should be Kurdish pops in.

**Comment by settintotrieste** [comment]
@Outta-my-way-burrocrat Sure, feel free to use them.

**Comment by Outta-my-way-burrocrat** [comment]
The Mahabad province is totally Azeri in 1946, which isn't accurate and makes it harder for the player. The concept of Kurdish nationalism wasn't overwhelmingly popular with the Kurds then, but there should be at least a few officer and soldier pops. As Mouseasel pointed out in issue #620 the Khvoy province should be controlled by South Azerbaijan in 1946.

Thank you @settintotrieste

**Comment by Outta-my-way-burrocrat** [comment]
[KRI_oob.txt](https://github.com/settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE/files/8637822/KRI_oob.txt)
What issues have you had setting up oobs?

**Comment by ccfox** [comment]
Mainly balance.

**Comment by DoktorKure** [comment]
Maybe a little late, but is it possible to change KRQ tag to KRG? The latter acronym is already used to refer to Iraqi Kurdistan in the form of the Kurdistan Regional Government.

**Comment by ccfox** [comment]
@DoktorKure The problem is that there could potentially be multiple regional Kurdish governments (given that there are/were proposals for an Autonomous regional government in Iran, Turkey, and Syria). I chose KRQ and KRI because that is where they will predominantly be, it also makes it easier for people to recognize which Kurdistan it is without too much fuss.

**Comment by Outta-my-way-burrocrat** [comment]
Could it have been caused by #893 ?

**Comment by Outta-my-way-burrocrat** [comment]
You need to pull the master branch then fix merge conflicts in Chrisfox's events.csv using this method: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line

File: CWE/history/war/Iran_crisis_of_1946.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#Iran_crisis_of_1946.txt
```
Comment (Outta-my-way-burrocrat) [inline CWE/history/war/Iran_crisis_of_1946.txt:1]: Typo in folder name.

Comment (ccfox) [reply CWE/history/war/Iran_crisis_of_1946.txt]: Thanks.

**Comment by Outta-my-way-burrocrat** [comment]
The countries.txt and Chrisfox's events.csv files are conflicting. The conflict can be solved using the command line.

**Comment by ccfox** [comment]
Yeah, I don't have github desktop - so instead once the event tweaks are pushed I'm just going to close the merge request, make a new branch, and then remake the branch. It turns out not even github knows why conflicts happen sometimes and not others and so making a new branch just resolves all conflicts as long as the initial state of the branch itself was up to date.

**Comment by ccfox** [comment]
Its not a rework, its just making new nations and tweaking events to keep wonkiness from happening. And no, but I am focusing on work ahead of mod stuff. Life gets busy sometimes. There are a fair amount of events I have to go through to ensure that everything works correctly.

**Comment by Outta-my-way-burrocrat** [comment]
@ccfox may I PR a conflict-free fork of your branch? https://github.com/Outta-my-way-burrocrat/Victoria-2-Cold-War-Enhancement-Mod-CWE/tree/Kurdistan-Tweaks-conflict-fix

**Comment by ccfox** [comment]
You can, sure.

**Pull request closed without merging**